src/helpers/getCountryFields.ts

```diff
@@ -1,21 +1,46 @@
-import { TGeography } from "@/types";
+import { TCountry, TGeography } from "@/types";
 
-const findCountryObject = (countrySearch: string, dataSet: TGeography[]) => {
+// Old
+const findCountryObjectOld = (countrySearch: string, dataSet: TGeography[]) => {
   let country = dataSet.find((c) => c.value.toLowerCase() === countrySearch.toLowerCase());
-  if (!country) country = dataSet.find((c) => c.display_value.toLowerCase() === countrySearch.toLowerCase());
   if (!country) country = dataSet.find((c) => c.slug.toLowerCase() === countrySearch.toLowerCase());
+  if (!country) country = dataSet.find((c) => c.display_value.toLowerCase() === countrySearch.toLowerCase());
+  if (!country) return null;
+  return country;
+};
+
+export const getCountrySlugOld = (countrySearch: string, dataSet: TGeography[]) => {
+  return findCountryObjectOld(countrySearch, dataSet)?.slug;
+};
+
+export const getCountryNameOld = (countrySearch: string, dataSet: TGeography[]) => {
+  return findCountryObjectOld(countrySearch, dataSet)?.display_value;
+};
+
+export const getCountryCodeOld = (countrySearch: string, dataSet: TGeography[]) => {
+  return findCountryObjectOld(countrySearch, dataSet)?.value;
+};
+
+// New geographies-api
+const findCountryObject = (countrySearch: string, dataSet: TCountry[]) => {
```

Comment:
non-blocking: Is this a little clearer? Not 

```suggestion
const findCountry = (countrySearch: string, dataSet: TCountry[]) => {
```